src/main/java/com/codice/acdebugger/breakpoints/ImpliesBreakpointProcessor.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.codice.acdebugger.breakpoints;
+
+import com.codice.acdebugger.impl.BreakpointInfo;
+import com.codice.acdebugger.impl.BreakpointLocation;
+import com.codice.acdebugger.api.BreakpointProcessor;
+import com.codice.acdebugger.api.Debug;
+import com.google.common.collect.ImmutableSet;
+import com.sun.jdi.ObjectReference;
+import com.sun.jdi.StackFrame;
+import com.sun.jdi.ThreadReference;
+import com.sun.jdi.request.EventRequest;
+import java.security.Permission;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Old version of a breakpoint processor that was attempting to intercept all calls to {@link
+ * java.security.ProtectionDomain#implies(Permission)}.
+ */
+public class ImpliesBreakpointProcessor implements BreakpointProcessor {
+  private static final Set<BreakpointLocation> LOCATIONS =
+      ImmutableSet.of(
+          BreakpointProcessor.createLocationFor("Ljava/security/ProtectionDomain;", 290),
+          BreakpointProcessor.createLocationFor("Ljava/security/Permissions;", 185),
+          BreakpointProcessor.createLocationFor(
+              "Lorg/eclipse/osgi/internal/permadmin/SecurityAdmin;", 131),
+          BreakpointProcessor.createLocationFor(
+              "Lorg/eclipse/osgi/internal/permadmin/SecurityAdmin;", 134) /*,
+        BreakpointLocation.createLocationFor("Lorg/eclipse/osgi/internal/permadmin/BundlePermissions;", 86)*/);
```

Comment:
Why is this commented out?

Reply:
Because this whole class is outdated and currently not used. It is just left as a starting point for future changes